web/src/components/KebabFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { MAIN_DISCLAIMER_ROUTE } from 'api-client/src'
+
+import buildConfig from '../constants/buildConfig'
+import { RoutePatterns } from '../routes'
+import Footer from './Footer'
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: ${props => props.theme.colors.textColor};
+  width: 100%;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0 20px;
+  border-bottom: 1px solid ${props => props.theme.colors.textColor};
+  &:first-child {
+    margin-top: -10px;
+  }
+  &:last-child {
+    border-bottom: none;
+  }
+`
+
+type PropsType = {
+  language: string
+}
+
+const KebabFooter = ({ language }: PropsType): ReactElement => {
```

Comment:
I think we should not implement everything twice. Instead it would be nice if we could reuse the `LocationFooter`, for example by just passing a prop `inSidebar` or similar and applying the necessary styling based on that.